modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -399,9 +401,11 @@ export default {
       this.docFields.data = newInstance;
       this.prepErrors();
       this.docReady = true;
+      this.modal.triggerFocusRefresh++;
     } else {
-      this.$nextTick(() => {
-        this.loadNewInstance();
+      this.$nextTick(async () => {
+        await this.loadNewInstance();
+        this.modal.triggerFocusRefresh++;
```

Comment:
refresh focus elements in the modal, to take into account elements to focus from the new instance